src/qtgui/bookmarks.cpp

```diff
@@ -289,6 +289,32 @@ bool Bookmarks::removeTag(QString tagName)
     return true;
 }
 
+bool Bookmarks::renameTag(QString oldName, QString newName)
+{
+    oldName = oldName.trimmed();
+    newName = newName.trimmed();
+
+    if(oldName == TagInfo::strUntagged || oldName == "")
+        return false;
+
+    if (oldName == newName)
+        return false;
```

Comment:
Lines 300..310 can be removed. I missed the same check at the line 254.
Or it may be better to remove check on line 254 as it looks better to compare trimmed strings.